src/components/Modal/index.tsx

```diff
@@ -100,21 +101,31 @@ class Modal extends React.Component<ThemedModalProps, any> {
 		const { width, theme, ...props } = this.props;
 
 		const cancelButtonProps = Object.assign(
-			{ style: { marginRight: 20 } },
+			{ ml: 20 },
```

Comment:
Let's set it to `3` instead for consistency

Reply:
@thgreasi if you don't wanna rerun everything, feel free to self-certify as it's not a big deal

Reply:
@sradevski does self-certifying allow you to bypass screenshot testing?

Reply:
no :)

Reply:
Oh ok I now got what you mean :)

Reply:
Actually let me open a new PR on top of this.
See: https://github.com/balena-io-modules/rendition/pull/1214